cmd/kops/toolbox.go

```diff
@@ -18,14 +18,18 @@ package main
 
```

Comment:
It's a good place to start I think; I think it's reasonable to move it out of the toolbox later (I feel that is more acceptable in the toolbox than elsewhere)